### PR TITLE
Show enabled game modes for maps in map catalog

### PR DESCRIPTION
**Pull request by BrownHedgehog78**

### CODE REVIEW

**Review by RyanZem — APPROVED** [review]

File: mods/ctf/ctf_modebase/map_catalog.lua [hunk 1]
```diff
@@ -104,6 +104,19 @@ local function show_catalog(pname, current_map)
 		y = y + 1.5
 	end
 
+	if current_map_meta.game_modes and next(current_map_meta.game_modes) then
```
Comment (savilli) [inline mods/ctf/ctf_modebase/map_catalog.lua:107]: #current_map_meta.game_modes > 0

File: mods/ctf/ctf_modebase/map_catalog.lua [hunk 2]
```diff
@@ -104,6 +104,19 @@ local function show_catalog(pname, current_map)
...
+		table.sort(current_map_meta.game_modes)
+		formspec.elements.game_modes = {
+			type = "textarea",
+			pos = {7, y},
+			size = {10, 3},
+			label = minetest.colorize("#ffff00", "GAME MODES"),
+			read_only = true,
+			default = table.concat(current_map_meta.game_modes, ", "),
```
Comment (savilli) [inline mods/ctf/ctf_modebase/map_catalog.lua:115]: should be HumanReadable there

Comment (BrownHedgehog78) [reply mods/ctf/ctf_modebase/map_catalog.lua:115]: The game mode names?

File: mods/ctf/ctf_modebase/map_catalog.lua [hunk 3]
```diff
@@ -104,6 +104,19 @@ local function show_catalog(pname, current_map)
...
 
+	if current_map_meta.game_modes and #current_map_meta.game_modes > 0 then
+		table.sort(current_map_meta.game_modes)
+		formspec.elements.game_modes = {
+			type = "textarea",
+			pos = {7, y},
+			size = {10, 3},
+			label = minetest.colorize("#ffff00", "GAME MODES"),
+			read_only = true,
+			default = HumanReadable(table.concat(current_map_meta.game_modes, ", ")),
```
Comment (savilli) [inline mods/ctf/ctf_modebase/map_catalog.lua:115]: You can just do `HumanReadable(current_map_meta.game_modes)`. HumanReadable does sorting as well.

Comment (BrownHedgehog78) [reply mods/ctf/ctf_modebase/map_catalog.lua]: Done

File: mods/ctf/ctf_modebase/map_catalog.lua [hunk 4]
```diff
@@ -104,6 +104,18 @@ local function show_catalog(pname, current_map)
...
 	end
 
+	if current_map_meta.game_modes and #current_map_meta.game_modes > 0 then
+		formspec.elements.game_modes = {
+			type = "textarea",
+			pos = {7, y},
+			size = {10, 3},
+			label = minetest.colorize("#ffff00", "GAME MODES"),
+			read_only = true,
+			default = HumanReadable(current_map_meta.game_modes, ", "),
```
Comment (savilli) [inline mods/ctf/ctf_modebase/map_catalog.lua:114]: HumanReadable has only one arg

Comment (BrownHedgehog78) [reply mods/ctf/ctf_modebase/map_catalog.lua:114]: my mistake

**Pull request merged**